src/main/java/frc/robot/RobotContainer.java

```diff
@@ -31,7 +31,9 @@
 import frc.robot.subsystems.indexer.Indexer;
 import frc.robot.subsystems.indexer.IndexerIO;
 import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.intake.IntakeConstants;
 import frc.robot.subsystems.intake.IntakeIO;
+import frc.robot.subsystems.intake.IntakeIOPhysical;
```

Comment:
remove these imports, besides line 35 (IntakeIO), after you remove all requested references of intake in this file